packages/tokens/config.js

```diff
@@ -83,6 +83,14 @@ module.exports = {
           destination: 'js/themes.js',
           format: 'javascript/es6/vads-colors-themes',
         },
+        {
+          destination: 'js/font/styles.js',
```

Comment:
Should be prior to themes/spacing for alphabetical ordering based on destination.